### PR TITLE
tested pipeline samples

### PR DESCRIPTION
**Pull request by animeshsingh**
add the list of currently tested pipelines, and reorient the main readme pointing to implementation details and test results, both compilation tests and execution tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,32 +12,45 @@ of a Pipeline and is responsible for creating a Pod for each of its Tasks and as
...
+### Developer Guide
+[Developer Guide](/sdk/python/README.md) 
 
-We are currently using [Kubeflow Pipelines 0.5.0](https://github.com/kubeflow/pipelines/releases/tag/0.5.0) and
-[Tekton 0.11.3](https://github.com/tektoncd/pipeline/releases/tag/v0.11.3) for this project. 
+### Available Features and Implementation Details
+[Available Features and Implementation Details](/sdk/FEATURES.md)
+
+### Compiler Status Report
+[Compilation Status Test Report](/tests/README.md)
```
Comment (Tomcli) [inline README.md:36]:
```suggestion
[Compilation Status Test Report](/sdk/python/tests/README.md)
```

Comment (animeshsingh) [reply README.md]: fixed @Tomcli

File: samples/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# KFP Tekton Samples
+
+Below are the list of samples that are currently running end to end taking the compiled Tekton yaml and deploying on a Tekton cluster directly. 
+If you are interested more in the larger list of pipelines samples we are testing for whether they can be 'compiled to Tekton' format, please [look at the corresponding status page](/sdk//python/tests/README.md)
```
Comment (kevinyu98) [inline samples/README.md:4]: remove the extra slash. (/sdk/python/tests/README.md)

Comment (Tomcli) [reply samples/README.md:4]: The extra slash won't affect the link, but probably is nice to just have single slash to avoid confusion.

**Pull request merged**